### PR TITLE
Church stat shifts

### PR DESCRIPTION
**Pull request by Onutsio**
Did you guys know Templar had the highest stat boon of the town other than Captain? Barely beating out the RG themselves. I learned this while wandering to myself "Wow, my acolyte stats sure feel dogwater!", and went looking.

Summary:
Acolyte-
Intelligence 1->3
Endurance 1->2
Speed 0->1
Removed per penalty
Acolytes, being the booky, educated types of the Church, now are actually smarter than their lumpheaded brethren. The endurance tick helps them cast more miracles, and the Speed bonus hastens them to their needed places, while not being so fast as to make them dodge gamers(The Dark hand of combat creep will NOT take me)

Templar
Str, Con, and End remain the same
Perception/Intelligent bonuses removed
Spd malus increased to -2.
This puts them roughly on par with MAA, being a bit stronger, but slower. They also still have the bonus of being cleric casters.

## Why It's Good For The Game

Makes Acolytes stats not dog. Severely reigns in Templar, who were for some reason good at everything. I'm of the opinion that the garrison/town combat roles should be toned down overall, but that's another conversation. I greatly resisted the urge to turn Acolytes into Karate Monks with dodge expert as the Dark Hand of Combat-role creep tried to take me.

### CODE REVIEW

**Comment by bearrrrrrrr** [comment]
seems fine tbh
a lot of our roles are insanely powercrept. makes sense- we fucked w/ adventurers for being insanely powercrept, now we gotta do the same to toher roles
UGHH

**Comment by Kavrick** [comment]
This should be expanded to a lot of other roles imo. Pretty much just every non-combat role gets worse stats than combat roles, even in non-combat stats, which is incredibly stupid. It just encourages more people to play combat roles rather than other roles.

I might go through and buff some of the others, I think we need a 'guide' on how many total stat bonuses different types of roles give.

Imo it should go 'roles without responsibility' < 'roles with responsibility' and then 'leadership roles' when it comes to determining how many stat increases they get. maybe something like 4/5/6.

**Comment by bearrrrrrrr** [comment]
> This should be expanded to a lot of other roles imo. Pretty much just every non-combat role gets worse stats than combat roles, even in non-combat stats, which is incredibly stupid. It just encourages more people to play combat roles rather than other roles.
> 
> I might go through and buff some of the others, I think we need a 'guide' on how many total stat bonuses different types of roles give.
> 
> Imo it should go 'roles without responsibility' < 'roles with responsibility' and then 'leadership roles' when it comes to determining how many stat increases they get. maybe something like 4/5/6.



> This should be expanded to a lot of other roles imo. Pretty much just every non-combat role gets worse stats than combat roles, even in non-combat stats, which is incredibly stupid. It just encourages more people to play combat roles rather than other roles.
> 
> I might go through and buff some of the others, I think we need a 'guide' on how many total stat bonuses different types of roles give.
> 
> Imo it should go 'roles without responsibility' < 'roles with responsibility' and then 'leadership roles' when it comes to determining how many stat increases they get. maybe something like 4/5/6.

yeah agreed

we definitely need to actually have a webmd design doc for this actually LOL

**Comment by worriedconcentrate** [comment]
So I'm going to start this off by saying, hey Onutsio thanks for coding, you're awesome! As for the acolyte buffs, they are completely worthless and make little to no sense for the fluff of the role. They are still not designed to participate in fights, as they should be, they are priests not warriors. Buffing non-combat classes.. save for MAYBE hunter which is just a strictly worse ranger, is silly and I think is against the entire spirit of the codebase. Non-combat roles should stay that way and I cannot fathom why there is even a discussion about buffing them or balancing them against combat roles.

Now for the templar nerfs, a minus one speed nerf is fair enough. The templar is a combat carry role for the church, like the royal guard is for the keep. Comparing them to a secondary spam combat role like men-at-arms is an odd thing to do but fair enough I suppose. The nerfs to intelligence and perception are completely and utterly pointless, they do not affect anything but fluff for the class, BUT maybe if you wanted to to take a statpack and grind crafting for a long time you could do something ranged with them but.. eh.. 

TL;DR Acolyte buffs are silly and won’t actually make people play them more and dropping four points of worthless stats from a melee slugger is silly and isn’t a real nerf.

**Comment by bearrrrrrrr** [comment]
> Buffing non-combat classes.. save for MAYBE hunter which is just a strictly worse ranger, is silly and I think is against the entire spirit of the codebase.

This is literally why people death-stacked adventurer.

**Comment by MSieghardt** [comment]
It makes 100% sense for Acolytes to have higher intelligence given they're the scholarly type. Consider the fact that having better stats helps a person want to play more. Mechanics and roleplay go hand in hand. Good change.

**Comment by Onutsio** [comment]
> So I'm going to start this off by saying, hey Onutsio thanks for coding, you're awesome! As for the acolyte buffs, they are completely worthless and make little to no sense for the fluff of the role. They are still not designed to participate in fights, as they should be, they are priests not warriors. Buffing non-combat classes.. save for MAYBE hunter which is just a strictly worse ranger, is silly and I think is against the entire spirit of the codebase. Non-combat roles should stay that way and I cannot fathom why there is even a discussion about buffing them or balancing them against combat roles.
> 
> Now for the templar nerfs, a minus one speed nerf is fair enough. The templar is a combat carry role for the church, like the royal guard is for the keep. Comparing them to a secondary spam combat role like men-at-arms is an odd thing to do but fair enough I suppose. The nerfs to intelligence and perception are completely and utterly pointless, they do not affect anything but fluff for the class, BUT maybe if you wanted to to take a statpack and grind crafting for a long time you could do something ranged with them but.. eh..
> 
> TL;DR Acolyte buffs are silly and won’t actually make people play them more and dropping four points of worthless stats from a melee slugger is silly and isn’t a real nerf.

Im a bit confused? I didnt buff Acolytes combat skills or stats. Int is to help with crafting, end helps them miracle more, and the +1pt of spd is just to move a bit.  The stats given, esp int, are fluffy and just help the role play their support role a bit more. 

We dont need 3 RG more in Templar. Especially since Templar are basically just a religious garrison role, with how rarely church/crown conflict.  I do agree that this isnt much of a combat nerf to Templar. I think that should come when every town combat role eats a stat nerf
